src/qmapshack/poi/CPoiPOI_TagMap.cpp

```diff
@@ -16,266 +16,352 @@
+    map["shop=photo"] = CPoiIconCategory(QPixmap("://icons/poi/SJJB/png/shopping_photo.n.32.png"));
+    map["shop=seafood"] = CPoiIconCategory(QPixmap("://icons/poi/SJJB/png/shopping_fish.n.32.png"));
+    map["shop=supermarket"] = CPoiIconCategory(QPixmap("://icons/poi/SJJB/png/shopping_supermarket.n.32.png"));
+    map["shop=tobacco"] = CPoiIconCategory(QPixmap("://icons/poi/SJJB/png/shopping_tobacco.n.32.png"));
+    map["shop=toys"] = CPoiIconCategory(QPixmap("://icons/poi/SJJB/png/shopping_toys.n.32.png"));
+    map["shop=video"] = CPoiIconCategory(QPixmap("://icons/poi/SJJB/png/shopping_video_rental.n.32.png"));
+    map["sport=archery"] = CPoiIconCategory(QPixmap("://icons/poi/SJJB/png/sport_archery.n.32.png"));
+    map["sport=baseball"] = CPoiIconCategory(QPixmap("://icons/poi/SJJB/png/sport_baseball.n.32.png"));
+    map["sport=canoe"] = CPoiIconCategory(QPixmap("://icons/poi/SJJB/png/sport_canoe.n.32.png"));
+    map["sport=cliff_diving"] = CPoiIconCategory(QPixmap("://icons/poi/SJJB/png/sport_diving.n.32.png"));
+    map["sport=climbing"] = CPoiIconCategory(QPixmap("://icons/poi/png/sport_rock_climbing_n32.png"));
+    map["sport=cricket"] = CPoiIconCategory(QPixmap("://icons/poi/SJJB/png/sport_cricket.n.32.png"));
+    map["sport=horse_reacing"] = CPoiIconCategory(QPixmap("://icons/poi/SJJB/png/sport_horse_racing.n.32.png"));
+    map["sport=ice_skating"] = CPoiIconCategory(QPixmap("://icons/poi/SJJB/png/sport_iceskating.n.32.png"));
+    map["sport=miniature_golf"] = CPoiIconCategory(QPixmap("://icons/poi/SJJB/png/sport_minature_golf.n.32.png"));
+    map["sport=motor"] = CPoiIconCategory(QPixmap("://icons/poi/SJJB/png/sport_motorracing.n.32.png"));
+    map["sport=sailing"] = CPoiIconCategory(QPixmap("://icons/poi/SJJB/png/sport_sailing.n.32.png"));
+    map["sport=scuba_diving"] = CPoiIconCategory(QPixmap("://icons/poi/png/sport_scuba_diving_n32.png"));
+    map["sport=shooting"] = CPoiIconCategory(QPixmap("://icons/poi/SJJB/png/sport_shooting.n.32.png"));
+    map["sport=skiing"] = CPoiIconCategory(QPixmap("://icons/poi/SJJB/png/sport_skiing_downhill.n.32.png"));
+    map["sport=snooker"] = CPoiIconCategory(QPixmap("://icons/poi/SJJB/png/sport_snooker.n.32.png"));
+    map["sport=soccer"] = CPoiIconCategory(QPixmap("://icons/poi/SJJB/png/sport_soccer.n.32.png"));
+    map["sport=surfing"] = CPoiIconCategory(QPixmap("://icons/poi/SJJB/png/sport_windsurfing.n.32.png"));
+    map["sport=swimming"] = CPoiIconCategory(QPixmap("://icons/poi/SJJB/png/sport_swimming_outdoor.n.32.png"),
+    {
+        {"building=yes", QPixmap("://icons/poi/SJJB/png/sport_swimming_indoor.n.32.png")}
+    });
+    map["sport=tennis"] = CPoiIconCategory(QPixmap("://icons/poi/SJJB/png/sport_tennis.n.32.png"));
+    map["station=subway"] = CPoiIconCategory(QPixmap("://icons/poi/SJJB/png/transport_subway.n.32.png"));
+    map["tourism=alpine_hut"] = CPoiIconCategory(QPixmap("://icons/poi/SJJB/png/accommodation_alpinehut_r_n32.png"));
```

Comment:
The icon is not found in  icons/poi/SJJB/png/ but in icons/poi/png/  ,  so for now  the icon is not rendered  and a white square appears in place

`map["tourism=alpine_hut"] = CPoiIconCategory(QPixmap("://icons/poi/png/accommodation_alpinehut_r_n32.png"));` fixes it

Reply:
Thanks!